### PR TITLE
My_portfolio: Milestone 3

### PR DESCRIPTION
**Pull request by Donmark2k**
In this milestone;

1. We checkout of the main branch
2. created a new branch, created_contactme
3.  I created a contact form
4. my partner styled it
5. I updated the readme

### CODE REVIEW

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# INVALID REVIEW REQUEST

Hi @Donmark2k , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have linter configurations, and the README file is not edited Kindly update it and ask for  review again

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -1,174 +1,285 @@
...
+            <textarea
+              id="msg"
+              name="user_message"
+              maxlength="500"
+              required
+              placeholder="Enter text here"
+            ></textarea>
+          </li>
+          <li>
+            <input type="submit" value="Get in touch" class="btn buttons" />
```
Comment (Lizdev-05) [inline index.html:253]: [OPTIONAL] Great job so far, I don't think there is any need for using the `ul` tag here. As programmers, one of our duties is to keep our code dry.

Comment (Lizdev-05) [inline index.html:253]: Kindly add interaction such as a hover effect to your button when clicked just like what we have in the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=W3PwpACp8rxJFwSN-0)👇
Screenshot 2022-12-06 at 21 09 14

**Review by Lizdev-05 — CHANGES_REQUESTED** [review]
Hi @Donmark2k, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Correct GitHub Flow 👍

- Good commit history 💰

- Good PR title/description 💯

- All linters passed 🏅

- Personalised README file 👍
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- Kindly check the live link provided on the README, it is not working😎
- Also, kindly check the image link on the README, seems like the link is broken 😊

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Donmark2k) [reply index.html:253]: the hover button is working as indicated on the figma, it was meant to drop a shadow effect on hover.

Comment (Donmark2k) [reply index.html:253]: please explain these in a clearer form, Should I have used ul without li or I should not use ul in the first place.

**Review by Alfredbis29 — APPROVED** [review]
# STATUS: APPROVED  
 ## Hi  @Donmark2k  
Your project is complete! There is nothing else to say other than... Great job clap Congratulations! :tada:
 
 TO HIGHLIGHT:
 
 * Your project is professional :white_check_mark:
 * Your repo is professional :white_check_mark:
 * Well structured files :white_check_mark:
 * No Linter errors :white_check_mark:
 * use new readme template provided by microverse :white_check_mark:
 
 Now you can merge this branch and move on :1st_place_medal: HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers four_leaf_clover:


 
 ## Optional suggestions
- [ ]  N/A

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 
Cheers and Happy coding! :clap:  :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**